2024-03-15 14:36:13,875 INFO    MainThread:14900 [wandb_setup.py:_flush():76] Current SDK version is 0.16.4
2024-03-15 14:36:13,875 INFO    MainThread:14900 [wandb_setup.py:_flush():76] Configure stats pid to 14900
2024-03-15 14:36:13,875 INFO    MainThread:14900 [wandb_setup.py:_flush():76] Loading settings from C:\Users\USER\.config\wandb\settings
2024-03-15 14:36:13,875 INFO    MainThread:14900 [wandb_setup.py:_flush():76] Loading settings from C:\Users\USER\Desktop\mini-marl\vdn\wandb\settings
2024-03-15 14:36:13,875 INFO    MainThread:14900 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-03-15 14:36:13,875 INFO    MainThread:14900 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-03-15 14:36:13,876 INFO    MainThread:14900 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'vdn\\main.py', 'program_abspath': 'C:\\Users\\USER\\Desktop\\mini-marl\\vdn\\main.py', 'program': 'C:\\Users\\USER\\Desktop\\mini-marl\\vdn\\main.py'}
2024-03-15 14:36:13,876 INFO    MainThread:14900 [wandb_init.py:_log_setup():526] Logging user logs to C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_143613-uoic292g\logs\debug.log
2024-03-15 14:36:13,877 INFO    MainThread:14900 [wandb_init.py:_log_setup():527] Logging internal logs to C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_143613-uoic292g\logs\debug-internal.log
2024-03-15 14:36:13,877 INFO    MainThread:14900 [wandb_init.py:init():566] calling init triggers
2024-03-15 14:36:13,877 INFO    MainThread:14900 [wandb_init.py:init():573] wandb.init called with sweep_config: {}
config: {'use_wandb': True, 'entity_name': 'singfor7012', 'env_name': 'ma_gym:Checkers-v0', 'experiment_name': 'vdn', 'use_cuda': False, 'n_training_threads': 12, 'use_render': False, 'use_time_sleep': False, 'sleep_second': 0.5, 'seed': 42, 'fix_seed': True, 'epsilon_anneal_episode': 60000, 'max_epsilon': 0.9, 'min_epsilon': 0.05, 'lr': 0.0001, 'gamma': 0.99, 'batch_size': 32, 'max_episodes': 100000, 'max_step': 100, 'step_cost': -0.01, 'chunk_size': 10, 'update_iter': 1, 'update_target_interval': 50, 'grad_clip_norm': 0.5, 'decide_target': 'double_dqn', 'decide_network': 'q_net', 'use_recurrent': True, 'test_interval': 10, 'test_episodes': 10, 'buffer_limit': 10000, 'eps': 1e-06, 'alpha': 0.4, 'beta': 0.4, 'update_alpha_beta': True}
2024-03-15 14:36:13,877 INFO    MainThread:14900 [wandb_init.py:init():616] starting backend
2024-03-15 14:36:13,877 INFO    MainThread:14900 [wandb_init.py:init():620] setting up manager
2024-03-15 14:36:13,886 INFO    MainThread:14900 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn, using: spawn
2024-03-15 14:36:13,889 INFO    MainThread:14900 [wandb_init.py:init():628] backend started and connected
2024-03-15 14:36:13,910 INFO    MainThread:14900 [wandb_init.py:init():720] updated telemetry
2024-03-15 14:36:13,982 INFO    MainThread:14900 [wandb_init.py:init():753] communicating run to backend with 90.0 second timeout
2024-03-15 14:36:14,546 INFO    MainThread:14900 [wandb_run.py:_on_init():2262] communicating current version
2024-03-15 14:36:14,662 INFO    MainThread:14900 [wandb_run.py:_on_init():2271] got version response 
2024-03-15 14:36:14,662 INFO    MainThread:14900 [wandb_init.py:init():804] starting run threads in backend
2024-03-15 14:36:16,691 WARNING MainThread:14900 [wandb_init.py:init():1198] interrupted
Traceback (most recent call last):
  File "C:\Users\USER\anaconda3\envs\torch\lib\site-packages\wandb\sdk\wandb_init.py", line 1176, in init
    run = wi.init()
  File "C:\Users\USER\anaconda3\envs\torch\lib\site-packages\wandb\sdk\wandb_init.py", line 817, in init
    run_start_result = run_start_handle.wait(timeout=30)
  File "C:\Users\USER\anaconda3\envs\torch\lib\site-packages\wandb\sdk\lib\mailbox.py", line 283, in wait
    found, abandoned = self._slot._get_and_clear(timeout=wait_timeout)
  File "C:\Users\USER\anaconda3\envs\torch\lib\site-packages\wandb\sdk\lib\mailbox.py", line 130, in _get_and_clear
    if self._wait(timeout=timeout):
  File "C:\Users\USER\anaconda3\envs\torch\lib\site-packages\wandb\sdk\lib\mailbox.py", line 126, in _wait
    return self._event.wait(timeout=timeout)
  File "C:\Users\USER\anaconda3\envs\torch\lib\threading.py", line 581, in wait
    signaled = self._cond.wait(timeout)
  File "C:\Users\USER\anaconda3\envs\torch\lib\threading.py", line 316, in wait
    gotit = waiter.acquire(True, timeout)
KeyboardInterrupt
2024-03-15 14:36:25,949 WARNING MsgRouterThr:14900 [router.py:message_loop():77] message_loop has been closed
